lib/selftest/src/Obelisk/SelfTest.hs

```diff
@@ -166,9 +165,6 @@ main = do
           it ("can enter "    <> shell) $ inTmpObInit $ \_ -> inShell "exit"
           it ("can build in " <> shell) $ inTmpObInit $ \_ -> inShell $ "cabal new-build --" <> fromString compiler <> " all"
 
-        it "can build reflex project" $ inTmpObInit $ \_ -> do
```

Comment:
Fails with either `error: Android builds are not supported on this platform.` or `error: iOS builds are not supported on this platform.`.

Reply:
Yeah this is not my favorite design that `nix-build` doesn't work without `-A`. Perhaps we can add a `-A all-supported-platforms` or something that just filters out anything that doesn't get through `tryEval`.

Reply:
Or something much less hacky...

Reply:
I think that would be a better solution that removing this test case. What do you think?